test-content/components/Header.test.js

```diff
@@ -1,6 +1,8 @@
+/* globals beforeEach, afterEach, describe, it */
+
 const assert = require("chai").assert;
 const Header = require("components/Header/Header");
-const React = require("react");
+const React = require("react"); // eslint-disable-line no-unused-vars
```

Comment:
Q: what does eslinit complain about here? It's not like this is doing anything different from the lines above/below?